main.go

```diff
@@ -114,13 +114,18 @@ func ReportSubcommand() {
 }
 
 func main() {
-	// TODO(#6): index out of range error when no subcommands are provided
-	switch os.Args[1] {
-	case "list":
-		ListSubcommand()
-	case "report":
-		ReportSubcommand()
-	default:
-		panic(fmt.Sprintf("`%s` unknown command", os.Args[1]))
+	if len(os.Args) > 1 {
+		switch os.Args[1] {
+		case "list":
+			ListSubcommand()
+		case "report":
+			ReportSubcommand()
+		default:
+			panic(fmt.Sprintf("`%s` unknown command", os.Args[1]))
+		}
+	} else {
+		//TODO (#7) implement a map for options instead of println'ing them all there
+		//also, not sure these descriptions are exactly what rexim means by them
+		fmt.Printf("snitch [opt]\n\tlist: lists all possible subcommands\n\treport: reports an issue to github\n")
```

Comment:
Doesn't Go have multiline strings? ![](https://static-cdn.jtvnw.net/emoticons/v1/425618/1.0)

Reply:
It does, but do note that multiline strings take their indent from code, so maintaining gofmt, it would result looking like [this](https://imgur.com/a/7ObWa5H) instead of [this](https://imgur.com/a/pgFdeKG) which looks nicer imo